src/applications/benefit-eligibility-questionnaire/pages/militaryService.js

```diff
@@ -20,6 +20,19 @@ export default {
         },
       },
     },
+    expectedSeparation: {
+      'ui:title': (
+        <>
+          <p>
+            <b>When do you expect to seperate or retire from the service?</b>
```

Comment:
Should "seperate" be spelled "separate"?

Reply:
Good catch! Thanks